actions/debootstrap_action.go

```diff
@@ -134,11 +134,11 @@ func (d *DebootstrapAction) Run(context *debos.DebosContext) error {
 	}
 
 	if d.Certificate != "" {
-		cmdline = append(cmdline, fmt.Sprintf("--certificate=%s", d.Certificate))
+		cmdline = append(cmdline, fmt.Sprintf("--certificate=%s", path.Join(context.Artifactdir, d.Certificate)))
```

Comment:
worth checking file exists first in `Verify` ?

Reply:
also, does this path need escaping or wrapping in quotes since it is an argument ?

Reply:
mmm... the escaping is needed for the shell. Since we are using os/exec -- arguments are passed to debootsrap as is. And with blank spaces it doesn't work since debootstrap script do not expect blank spaces :-(
`PRIVATEKEY="--private-key=$2"` <- this is the argument passed to wget as is.